### PR TITLE
Add logging for marketplace events

### PR DESCRIPTION
**Pull request by Realtin**

### CODE REVIEW

File: jobs/github-event/marketplace_purchase/cancelled.js [hunk 1]
```diff
@@ -1,11 +1,24 @@
...
 const dbs = require('../../../lib/dbs')
 const upsert = require('../../../lib/upsert')
 
 module.exports = async function ({ marketplace_purchase }) {
-  const { payments } = await dbs()
+  const { payments, logs } = await dbs()
+  const log = Log({
+    logsDb: logs,
+    accountId: marketplace_purchase.account.id,
+    repoSlug: null,
```
Comment (janl) [inline jobs/github-event/marketplace_purchase/cancelled.js]: not really needed?

Comment (janl) [reply jobs/github-event/marketplace_purchase/cancelled.js]: but good for explicitness :D

File: jobs/github-event/marketplace_purchase/cancelled.js [hunk 2]
```diff
@@ -1,11 +1,24 @@
...
   const accountId = String(marketplace_purchase.account.id)
 
-  await upsert(payments, accountId, {
-    plan: 'free'
-  })
+  try {
+    await upsert(payments, accountId, { plan: 'free' })
+    log.success('database: paymentDoc was updated', { plan: 'free' })
+  } catch (error) {
+    log.error('error', { error })
```
Comment (janl) [inline jobs/github-event/marketplace_purchase/cancelled.js]: maybe add prefix like in `+    log.error('database: no paymentDoc created', { error })` below

File: jobs/github-event/marketplace_purchase/changed.js [hunk 3]
```diff
@@ -1,12 +1,26 @@
...
 
-  await upsert(payments, accountId, {
-    plan: normalizePlanName(marketplace_purchase.plan.name)
-  })
+  try {
+    const plan = normalizePlanName(marketplace_purchase.plan.name)
+    await upsert(payments, accountId, { plan })
+    log.success('database: paymentDoc was updated', { plan })
+  } catch (error) {
+    log.error('error', { error })
```
Comment (janl) [inline jobs/github-event/marketplace_purchase/changed.js]: maybe add prefix like in `+    log.error('database: no paymentDoc created', { error })` below

File: jobs/github-event/marketplace_purchase/purchased.js [hunk 4]
```diff
@@ -14,11 +22,14 @@ module.exports = async function ({ marketplace_purchase }) {
...
   try {
     paymentDoc = await payments.get(String(accountId))
+    log.success('database: paymentDoc created', { paymentDoc })
   } catch (error) {
+    log.error('database: no paymentDoc created', { error })
     if (error.status !== 404) throw error
   }
 
   if (paymentDoc && paymentDoc.stripeSubscriptionId) {
+    log.info('sheduled `cancel-stripe-subscription` job')
```
Comment (espy) [inline jobs/github-event/marketplace_purchase/purchased.js]: Spelling -> `scheduled`

**Pull request merged**